content/applications/marketing/events/sell_tickets.rst

```diff
@@ -1,68 +1,137 @@
-============
-Sell tickets
-============
+==================
+Sell event tickets
+==================
 
-Create custom ticket tiers (with various price points) for potential attendees to choose from,
-directly on the event template form, under the :guilabel:`Tickets` tab. Odoo simplifies the
-ticket-purchasing process by providing plenty of payment method options.
+Odoo *Events* provides users with the ability to create custom event tickets (and ticket tiers),
+with various price points.
+
+It *also* allows them to sell event tickets in two different ways: via standard sales orders, and
+online through an integrated website.
+
+Odoo also simplifies the ticket-purchasing process by providing plenty of payment method options.
+
+.. tip::
+   To learn more about how to create custom tickets (and ticket tiers) for events, check out the
+   :doc:`create_events` documentation.
 
 Configuration
 =============
 
-First, in order to enable the creation (and selling of) event tickets, go to
-:menuselection:`Configuration --> Settings`, then enable the :guilabel:`Tickets` and
-:guilabel:`Online Ticketing` features.
+In order to sell event tickets in Odoo, some settings must first be enabled.
 
-The :guilabel:`Tickets` feature allows tickets to be sold for an event.
+First, navigate to :menuselection:`Events app --> Configuration --> Settings`. In the
+:guilabel:`Registration` section, there are two different settings: :guilabel:`Tickets` and
+:guilabel:`Online Ticketing`.
 
-The :guilabel:`Online Tickets` feature allows for the sale of tickets to occur through the website.
+The :guilabel:`Tickets` setting allows users to sell event tickets with standard sales orders.
+
+The :guilabel:`Online Ticketing` setting allows users to sell event tickets online through their
+integrated Odoo website.
+
+To activate one (or both) of these settings, tick the checkbox beside the desired features, and
+click :guilabel:`Save` to implement those settings.
 
 .. note::
-   If these options are *not* enabled, a default :guilabel:`Register` button will be available for
-   free registrations.
+   If these options are *not* enabled, a default :guilabel:`Register` button becomes available for
+   visitors to interact with and procure free registrations to the event.
 
 .. image:: sell_tickets/events-settings-tickets.png
    :align: center
    :alt: View of the settings page for Odoo Events.
 
-Sell tickets through sales orders
-=================================
+With those settings enabled, Odoo automatically creates a new *Product Type* called, *Event Ticket*,
+which is accessible on every product form. Odoo also creates three event registration products (with
+the *Product Type* set to *Event Ticket*) that can be used or modified for event tickets.
+
+.. important::
+   When creating a new event registration product, the *Product Type* **must** be set to *Event
+   Ticket* in order for it to be selected in the *Product* column in the *Tickets*  tab on an event
+   form.
```

Comment:
Consider adding some specificity here because we're talking about two different forms in the same admonition block (product form vs. events form > tickets tab). Think we're missing some guis, as well? 🤔